sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlobBaseAPITest.groovy

```diff
@@ -428,7 +428,7 @@ class BlobBaseAPITest extends APISpec {
 
     @RequiredServiceVersion(clazz = BlobServiceVersion.class, min = "V2019_12_12")
     @Retry(count = 5, delay = 5, condition = { environment.testMode == TestMode.LIVE })
-    @PlaybackOnly(expiryTime = "2022-05-18")
+    @PlaybackOnly(expiryTime = "2022-08-18")
```

Comment:
@ibrahimrabab @rickle-msft @seanmcc-msft
For some reason this has not been working ...
Could you guys follow up and un-playback-only this test?